src/types/student/page-meta-dto-parameters.ts

```diff
@@ -0,0 +1,6 @@
+import { PageOptionsDto } from 'src/students/dto';
```

Comment:
musisz tu usunąć i zrobić typ, i sobie potem ten typ tutaj zaimportować. W `types` nie importujemy niczego z reszty aplikacji. Inaczej wywali błędy (to właśnie poprawiałem w ostatnim fixie)